### PR TITLE
Add some scripts to get started more quickly 

### PR DESCRIPTION
**Pull request by anaisbetts**
This PR adds two new commands to a new "scripts" folder:
1. `script/create NAME` - does the work of renaming the project and fixing up the files so that you don't have to
2. `script/build` - compiles the project without having to open an IDE

`script/bootstrap` is forthcoming, it will install Xamarin Studio on your machine and fix up the licenses for you, as well as `script/deploy`, which will deploy the app to any locally connected device

### CODE REVIEW

File: script/build [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+MDTOOL="/Applications/Xamarin Studio.app/Contents/MacOS/mdtool"
+
+
+if [ ! -f "$MDTOOL" ]; then
+	echo "*** You need to install Xamarin Studio ***"	
+	exit 1
+fi
+
+scriptdir=`dirname $BASH_SOURCE`
+find "$scriptdir/.." -type f -name "*.sln" -exec "$MDTOOL" build $@ "{}" \;
```
Comment (jspahrsummers) [inline script/build]: `$@` should always appear quoted.

**Comment by jspahrsummers** [comment]
`script/scaffold` might be more familiar than `script/create` for the target audience.

Also, we use `script/cibuild` for CI in GHfM, and that's the file that Janky looks for by default, IIRC.

**Comment by anaisbetts** [comment]
@jspahrsummers So, I didn't want to call this `cibuild` because I'm still contemplating how I could run the tests - I could just call it that for now

**Comment by jspahrsummers** [comment]
Wouldn't you want a headless build script to run the tests anyways? I don't see much benefit to splitting those up, at least for native apps where tests don't take a million years.

**Comment by anaisbetts** [comment]
> Wouldn't you want a headless build script to run the tests anyways?

I totally do, but the way that Xamarin tests work right now is that they're a test runner that runs _in the simulator_, which makes the automation story pretty :poop:. We'll have to ask Anuj about this one.

**Pull request merged**